library/core/src/ptr/alignment.rs

```diff
@@ -76,6 +80,8 @@ impl Alignment {
     #[unstable(feature = "ptr_alignment_type", issue = "102070")]
     #[rustc_const_unstable(feature = "ptr_alignment_type", issue = "102070")]
     #[inline]
+    #[requires(align > 0)]
```

Comment:
Are you planning to add harnesses for this new contract in a follow up PR?

Reply:
check_from_size_align_unchecked actually exercises this.

Reply:
No, it doesn't. Each harness can only verify one contract. The contract may be exercised if users decide to stub from them, with `stub_verified`.

Reply:
My empirical data here shows otherwise: when I changed the `requires` clause on `from_size_align_unchecked` then verification of harness failed for the above precondition was no longer guaranteed.

Reply:
Finally fixed in https://github.com/model-checking/verify-rust-std/pull/42.